### ISSUE TITLE
Missing local system calls when using dockerId option

### ISSUE THREAD

**Issue by eedugon** [elastic/support-diagnostics#358]
When `--dockerId` is used, the following interesting data is collected:
```
docker-info.txt
docker-ps-all.txt
```

And at container ID level:
```
docker-cpu-info.txt
docker-internal-top.txt
docker-logs.txt
docker-process-list.txt
docker-sysctl.txt
docker-top-threads.txt
docker-top.txt
docker-ulimit.txt
docker-uname.txt
```

This is great, but I guess we are missing also the usual calls to analyze the local system, like generic sysstat and iostat data, ifconfig report, etc... That data is relevant and important still.

If dockerId is intended to fetch only the already reported information, what is the workaround to make the diag to also run the local system calls? If I don't add `--dockerId` the diagnostic discards anyway the local system stats as for some reason it doesn't detect there's a local Elasticsearch instance running (as a docker).
